openkfw/Oscar
# #112-Postgis-setup

## Description

* added setup and docs for Postgis DB and PgAdmin UI

## Motivation and Context

* moving from MongoDB to PostgreSQL with Gis plugins which is widely used in map apps

## How has this been tested?

* locally running app using provided scripts
* copy new variables from .env_example to your .env also in minikube folder
* first try running app using `start.sh` script which will run current version with MongoDB
* run app using `start_postgis.sh` which will run app with Postgis and PgAdmin DB UI, app won't work as services aren't refactored for Postgres yet, but containers will be running
* you can open PgAdmin UI by running `pgadmin.sh` and then log in using development credentials from .env_example
* in PgAdmin expand servers on the left and under it there should be 'oscar' database and default 'postgres' database

## Screenshots (if appropriate):

## Types of changes




## Checklist:
